.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateTimePeriodExtractor.cs

```diff
@@ -148,9 +150,9 @@ private IEnumerable<Token> MergeDateWithTimePeriodSuffix(string text, List<Extra
         // Cases like "today after 2:00pm", "1/1/2015 before 2:00 in the afternoon"
         // Valid connector in English for Before include: "before", "no later than", "in advance of", "prior to", "earlier than", "sooner than", "by", "till", "until"...
         // Valid connector in English for After include: "after", "later than"
-        private bool IsValidConnectorForDateAndTimePeriod(string text, out int length, bool inPrefix)
+        private int GetValidConnectorForDateAndTimePeriod(string text, bool inPrefix)
         {
-            length = 0;
+            int length = -1;

```

Comment:
Please declare a constant for it, and use it here and in the return checks.

Reply:
done.